sdk/src/types/block/block_id.rs

```diff
@@ -34,7 +34,7 @@ pub struct BlockId {
 
 impl BlockId {
     /// The length of a [`BlockId`]
-    pub const LENGTH: usize = 40;
+    pub const LENGTH: usize = 36;
```

Comment:
Since when is this the case?

Reply:
Because slot index is part of the BlockId and it's now 4 bytes shorter

Reply:
Ah sure makes sense, wen TIP updated

Reply:
🤷🏻